modules/aws_mysql/tf_module/main.tf

```diff
@@ -0,0 +1,49 @@
+data "aws_security_group" "mysql_security_group" {
+  name = "opta-${var.env_name}-db-sg"
+}
+
+data "aws_kms_key" "main" {
+  key_id = "alias/opta-${var.env_name}"
+}
+
+resource "random_password" "mysql_password" {
+  length  = 20
+  special = false
+}
+
+resource "random_string" "db_name_hash" {
```

Comment:
would that make sense to make this name customizable by the user?

I wonder how it will work if the user would like to create 2 mysql DB and what would be the way for them to differentiate them? 

Also if they want to monitor the DB (for example in datadog or cloudwatch) would that be easy for them to differentiate their DB?

I might be missing something here, may be the layer_name would contain some relevant info?

Reply:
That's just a random 4 letter hash appended to the end of the Actual Identifier.
Example: 
Template: `"opta-${var.layer_name}-${var.module_name}-${random_string.db_name_hash.result}"`
Sample: `opta-mysql-ci-mysql-jl72`

Reply:
so module name can be used to identify their DB?

for example:
```
  - name: purchase
     type: mysql
  - name: inventory
     type: mysql
```

Reply:
> so module name can be used to identify their DB?
> 
> for example:
> 
> ```
>   - name: purchase
>     type: mysql
>   - name: inventory
>     type: mysql
> ```

Yes, that's how we have implemented this.
Although I'll check this and confirm.

Reply:
Works.